wishlist-backend/config/urls.py

```diff
@@ -34,10 +34,13 @@
 from django.urls import path
 
 import products.views
-
+from wishlist.views import WishlistAPI
 urlpatterns = [
```

Comment:
nit: you should add a new line to separate the code from the imports.

Reply:
Will take care of this ahead.